src/dash-adapter.js

```diff
@@ -217,7 +218,7 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
    */
   _removeBindings(): void {
     this._shaka.removeEventListener('adaptation', this._onAdaptation);
-    this._shaka.removeEventListener('error', this._onError);
+    this._shaka.removeEventListener('error', this._onError.bind(this));
```

Comment:
use arrow function `() => this._onError`

Reply:
@OrenMe - so i will change all of the bindings...